ska_conda/ska_builder.py

```diff
@@ -9,34 +9,55 @@
 
 class SkaBuilder(object):
 
-    def __init__(self, ska_root=None):
-        if ska_root is None:
-            ska_root = "/data/acis/ska3_pkg/"
-        self.ska_root = ska_root
-        self.ska_build_dir = os.path.join(self.ska_root, "builds")
-        self.ska_src_dir = os.path.join(self.ska_root, "src")
-        os.environ["SKA_TOP_SRC_DIR"] = self.ska_src_dir
+    def __init__(self, build_root='.', user='sot', git_repo_path='git@github.com:{user}/{name}.git'):
+        self.user = user
+        self.git_repo_path = git_repo_path
+        self.build_dir = os.path.abspath(os.path.join(build_root, "builds"))
+        self.src_dir = os.path.abspath(os.path.join(build_root, "src"))
+        os.environ["SKA_TOP_SRC_DIR"] = self.src_dir
```

Comment:
Not sure if we want to use this as a somewhat global environment variable, or pass it to conda build as an env in the subprocess.

Reply:
I think it doesn't matter.